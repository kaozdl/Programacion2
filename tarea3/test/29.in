# liberar_estructuras
liberar_estructuras
insertar_lista (13,abril) (27,abril) .
insertar_binario (14,c) (16,d) (13,t) (5,a) .
finitario (1,m) 1 .
finitario (21,a) 1 .
finitario (22,g) 2 .
finitario (321,b) 2 1 .
liberar_estructuras
insertar_lista (13,abril) (27,abril) .
insertar_binario (14,c) (16,d) (13,t) (5,a) .
finitario (1,m) 1 .
finitario (21,a) 1 .
finitario (22,g) 2 .
finitario (321,b) 2 1 .
Fin
