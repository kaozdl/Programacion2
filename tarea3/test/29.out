1>
2>3>(0,primero)(13,abril)(27,abril)
4>
--(13,t)
-(16,d)
(14,c)
-(5,a)
1
2
5>
0
(1,m)
6>(1,m)
(1,m)
1
(21,a)
7>(1,m)
(1,m)
(21,a)
2
(21,a)(22,g)
8>(1,m)
(1,m)
(21,a)(22,g)
2
(321,b)
9>10>(0,primero)(13,abril)(27,abril)
11>
--(13,t)
-(16,d)
(14,c)
-(5,a)
1
2
12>
0
(1,m)
13>(1,m)
(1,m)
1
(21,a)
14>(1,m)
(1,m)
(21,a)
2
(21,a)(22,g)
15>(1,m)
(1,m)
(21,a)(22,g)
2
(321,b)
16>Fin.
